src/pages/steps/AdditionalInformationStep/OrganizerStep.tsx

```diff
@@ -199,7 +202,10 @@ const OrganizerStep = ({
     setIsOrganizerAddModalVisible(false);
   };
 
-  const isUitpasOrganizer = Object.values(cardSystems).length > 0;
+  const hasUitpasCardSystems = Object.values(cardSystems).length > 0;
+
+  const showCardSystems =
```

Comment:
```suggestion
  const shouldShowCardSystems =
```

Maybe rename this const to not confuse it being a function